lib/mongodb.js

```diff
@@ -821,4 +831,6 @@ MongoDB.prototype.ping = function (cb) {
   }
 };
 
-
+MongoDB.prototype.idType = function(model) {
```

Comment:
Sorry I am confused with the function `idType`, does it "return the idType of a model" or "return the converted id value in a certain type"?

From the function definition it takes in the model as param then satisfies "return the idType of a model"
But from the usage like [`var oid = idType(idValue);`](https://github.com/wpjunior/loopback-connector-mongodb/blob/b474da06a06606b914d9d6ad87a860c9f33df714/lib/mongodb.js#L190) you want it to "return the converted id value in a certain type".

Could you elaborate more? Thanks :)